### ISSUE TITLE
bug: hamburger menu button has no keyboard focus state

### ISSUE THREAD

**Issue by miclgael** [miclgael/issues.michaelgale.dev#3]
## Steps to reproduce the issue

1. Visit https://www.michaelgale.dev/about
2. Give `Get In Contact` button focus.
3. Tab to next item with keyboard

## Expected result

A focus ring appears

## Actual result

No state change at all

## Device and system info

**What device were you using? (e.g. phone, laptop)**
Laptop

**What web browser were you using? (e.g. Chrome, Safari)**
Firefox

**If you were using assistive technology (e.g. VoiceOver, JAWS, NVDA etc.) please list:**
N/A

## Additional comments

If you know which [WCAG](https://www.w3.org/TR/WCAG/) or [ATAG](https://www.w3.org/TR/ATAG20/) criteria this violates, please indicate it here:
N/A
If you have any direct aXe tools documentation please also include this here:
N/A

![Screenshot of mg.dev homepage navigation showing the issue described](https://github.com/miclgael/issues/assets/4301358/e7fff56b-5edd-4309-aea0-ea1479e20eb7)

**Comment by miclgael** [comment]
Work is planned for Dec 15, 2023.
